launcher/ui/dialogs/ExportInstanceDialog.cpp

```diff
@@ -70,6 +70,7 @@ ExportInstanceDialog::ExportInstanceDialog(InstancePtr instance, QWidget* parent
     auto prefix = QDir(instance->instanceRoot()).relativeFilePath(instance->gameRoot());
     proxyModel->ignoreFilesWithPath().insert({ FS::PathCombine(prefix, "logs"), FS::PathCombine(prefix, "crash-reports") });
     proxyModel->ignoreFilesWithName().append({ ".DS_Store", "thumbs.db", "Thumbs.db" });
+    proxyModel->blockedPaths().insert({ FS::PathCombine(prefix, ".cache"), FS::PathCombine(prefix, ".fabric") });
```

Comment:
```suggestion
    proxyModel->ignoreFilesWithPath().insert({ FS::PathCombine(prefix, ".cache"), FS::PathCombine(prefix, ".fabric"), FS::PathCombine(prefix, ".quilt") });
```
quilt has a similar cache directory as fabric, so i figure it should be included. this also uses `m_ignoreFiles` like the other lines

Reply:
Quilt nowadays uses `.cache`, adding `.quilt` would add support for older Loader versions which makes sense though. The reason I used blocked paths here is that it allows the user to still select it in case they really want to instead of hiding them entirely, should that really be changed?

Reply:
> The reason I used blocked paths here is that it allows the user to still select it in case they really want to instead of hiding them entirely

on my end, using `ignoreFilesWithPath()` and `blockedPaths()` have the same result: `.cache` is not shown at all
![image](https://github.com/PrismLauncher/PrismLauncher/assets/48872998/7d77d6bd-9573-43f2-9fb4-44f31db68851)

there also aren't many cases where the cache *should* be exported, as it can be system specific

Reply:
I thought that was the reason there's two different ways of ignoring paths, and I also tested it originally: 
![image](https://github.com/PrismLauncher/PrismLauncher/assets/38926001/998e9888-8d55-4d90-9cc8-0107e48e2ca4)

Since your screenshot also doesn't include the `config` folder I assume you have not actually run this instance (?)

Reply:
`blockedPaths` deselects items - it doesn't hide them - so you probably didn't run the instance :upside_down_face:!

This is probably the best option - just a few things to keep in mind:
- if the user selects it, they will be exporting remapped game jars which isn't good :/
- this won't affect instances with existing `.packignore` files

Reply:
ah yeah you're right, just that it was a vanilla instance (i forgot .cache isn't made by default 😆)

imo though, i don't see much of a reason to keep the cache. you shouldn't be exporting remapped game jars, and we already precedent for system specific things such as the `logs` folder. i think it would still be better to put this in `m_ignoreFiles`

Reply:
Oh, I agree then